src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/repository/impl/ords/JJDisputeRepositoryImpl.java

```diff
@@ -80,8 +91,65 @@ public JJDispute saveAndFlush(JJDispute entity) {
 	}
 
 	@Override
-	public void setStatus(String ticketNumber, JJDisputeStatus jjDisputeStatus, String userName) {
-		throw new NotYetImplementedException();
+	public void setStatus(String ticketNumber, JJDisputeStatus jjDisputeStatus, String userName, String partId, Long courtAppearanceId) {
+		try {
+			JJDispute jjDispute = map(jjDisputeApi.v1JjDisputeGet(ticketNumber));
+			// For some reason ORDS returns a valid object but with null fields if no object is found.
+			if (jjDispute != null && !StringUtils.isBlank(jjDispute.getTicketNumber())) {
+
+				String seizedYn = null;
+				String adjudicatorPartId = null;
+				String aattCd = null;
+				String dattCd = null;
+				String staffPartId = null;
+
+				if (!CollectionUtils.isEmpty(jjDispute.getJjDisputeCourtAppearanceRoPs()) && courtAppearanceId != null && partId != null) {
+					// Get the associated jj dispute's courtAppearance by id
+					JJDisputeCourtAppearanceRoP courtAppearance = jjDispute.getJjDisputeCourtAppearanceRoPs().stream()
+							.filter(app -> app.getId() == courtAppearanceId)
+							.findAny()
+							.orElse(null);
+
+					// Populate fields required to update court appearance
+					seizedYn = courtAppearance.getJjSeized() != null ? courtAppearance.getJjSeized().toString() : null;
```

Comment:
Note, there will be an NPE here if courtAppearance is null (which line 111 will return). I'll fix this in my PR.

Reply:
Oh that's right. Sorry I missed that. Good catch! Thanks for fixing it.